test_recipes.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from recipe import Recipe
+
+
+class RecipeTest(unittest.TestCase):
+    """ Class performing unit testing for class Recipe"""
+    def setUp(self):
+        """Defining setUp() method that runs prior to each test."""
+        self.newCat = Recipe()
+
+    def test_category_registration(self):
+        """ Test to test function create category """
+        self.newCat.recipe_categories = {}
+        self.newCat.category_register("category_one", "waweru@gmail.com")
+        category_registration = self.newCat.category_register("category", "waweru@gmail.com")
```

Comment:
`self.newCat.category_register("category", "waweru@gmail.com")`is used more than once in your code. DRY! That is why the `setUp()` method is used.